docs/troubleshooting/troubleshooting.md

```diff
@@ -1,4 +1,5 @@
 ## FAQ and Troubleshooting
 
 1. My nodes in Kubernetes cluster were replaced and the cStor pools are pending. How can I fix this? See [Node Replacement](migrate_pool_by_migrating_disks.md).
-2. Pool manager pods are running but it is not reconciling the CSPC state. See [Bad Disk](pool_operations_hung.md)
\ No newline at end of file
+2. Pool manager pods are running but it is not reconciling the CSPC state. See [Bad Disk](pool_operations_hung.md)
+3. How to run cStor-CSI on rancher based clusters? See [Rancher Prerequisites](rancher_prerequisite.md)
```

Comment:
Adding the rancher doc link here too will be good?

Reply:
IMO Yes, if user didn't follow the prerequisites properly and if things are not working as expected he will look at available troubleshooting section.